src/main/java/practice/StreamPractice.java

```diff
@@ -59,7 +87,16 @@ public Double getOddNumsAverage(List<Integer> numbers) {
      * return the names of all cats whose owners are women from `femaleAge` years old inclusively.
      */
     public List<String> getCatsNames(List<Person> peopleList, int femaleAge) {
-        return Collections.emptyList();
+        Predicate<Person> peoplePredicate = person -> person.getAge() >= femaleAge
+                && person.getSex().equals(Person.Sex.WOMAN)
+                && !person.getCats().isEmpty();
```

Comment:
```suggestion
        Predicate<Person> peoplePredicate = person -> person.getAge() >= femaleAge
                && person.getSex().equals(Person.Sex.WOMAN);
```

I don't think we need this